<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_dos">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>116</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_parentesis_de">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>156</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <property name="shortcut">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_borrar_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 169, 98);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_resta">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>323</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_ocho">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>198</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_raiz">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>364</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>âˆš</string>
    </property>
    <property name="shortcut">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_dividir">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>280</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="shortcut">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cero">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>239</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_parentesis_iz">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>115</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <property name="shortcut">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_porcentaje">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>323</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="shortcut">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_uno">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>116</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_decimal">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>197</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="shortcut">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_igual">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>Enter, Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cuatro">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>157</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_suma">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>239</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="valor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 90 14pt &quot;Arial Black&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_nueve">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>198</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_salir">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 127);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_borrar_todo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>239</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 169, 98);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="shortcut">
     <string>Esc, Backspace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cinco">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>157</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_seis">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>157</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_tres">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>116</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_potencia">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>364</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <property name="shortcut">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_siete">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>198</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_multiplicar">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>280</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="valor2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 87 10pt &quot;Arial Black&quot;;color:rgb(202, 202, 202)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_borrar_todo</sender>
   <signal>clicked()</signal>
   <receiver>valor</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
